cosmos/dbt/graph.py

```diff
@@ -931,7 +931,7 @@ def load_from_dbt_manifest(self) -> None:
                     resource_type=DbtResourceType(node_dict["resource_type"]),
                     depends_on=node_dict.get("depends_on", {}).get("nodes", []),
                     file_path=self.execution_config.project_path / _normalize_path(node_dict["original_file_path"]),
-                    tags=node_dict["tags"],
+                    tags=node_dict.get("tags", []),
```

Comment:
```suggestion
                    tags=node_dict.get("tags") or [],
```
This is better when we expect tags to be a list, and it can happen that `tags` key is present but set to `null` / `None`, and then it won't be set to `[]` with the current implementation `node_dict.get("tags". [])`